src/audio-player-composable/audio-player-composable.tsx

```diff
@@ -101,7 +107,43 @@ export const AudioPlayerComposable = ({
       playStateIcon,
     };
   };
+  const getForwardButtonProps = ({
+    onClick: consumerOnClick,
+  }: {
+    onClick?: () => void;
+  }) => {
+    // All the internal logic for defining aria and icon to show
+
+    const ariaLabel = 'Fast forward for 10 seconds';
+    const onClick = () => {
+      if (consumerOnClick) consumerOnClick();
+      onClickForward();
+    };
+
+    return {
+      ariaLabel,
+
+      onClick,
+    };
+  };
+  const getReplayButtonProps = ({
+    onClick: consumerOnClick,
+  }: {
+    onClick?: () => void;
+  }) => {
+    // All the internal logic for defining aria and icon to show
+
+    const ariaLabel = 'Rewind 10 seconds';
+    const onClick = () => {
+      if (consumerOnClick) consumerOnClick();
+      onClickBackward();
+    };
 
+    return {
+      ariaLabel,
+      onClick,
```

Comment:
the icon should be here, the same way its created for getPlayPauseButtonProps
```suggestion
      'aria-label',
      onClick,
      children: <IconFilledForward10 />
```